examples/tutorials/advanced/websockets-example-MNIST-parallel/start_websocket_servers.py

```diff
@@ -17,49 +17,63 @@
     ),
 )
 
+
+if os.name == "nt":
+    python = "python"
+else:
+    python = "python" + sys.version[0:3]
+
 call_alice = [
-    "python",
-    "run_websocket_server.py",
+    python,
+    "../../../../run_websocket_server.py",
```

Comment:
Q: Isn't there a nicer way to do this? Maybe use the PATH variable? @karlhigley

Reply:
I'm sure there is, but I don't know it off the top of my head. Might be worth researching.

Reply:
@gmuraru @karlhigley  I know only getting current directory by os.getcwd() and then splitting at last 4 '\' to get the path, will that be right?

Reply:
On reflection: one way to solve this would to symlink the file into these directories and then remove `../../../../`.

Reply:
@karlhigley  I tried `FILE_PATH = os.path.abspath(__file__)
FILE_PATH = "/".join(PATH.split('/')[:-5])+"/run_websocket_server.py"
` and it works. So, is it good?

Reply:
🤷‍♂ 

All of the solutions we've come up with are pretty brittle.

Reply:
> @karlhigley I tried `FILE_PATH = os.path.abspath(__file__)`
> `FILE_PATH = "/".join(FILE_PATH.split("/")[:-5]) + "/run_websocket_server.py"` and it works. So, is it good?

@karlhigley My last commit is using this. Any other suggestions?

Reply:
why last solution proposed is brittle? where it will break?

Reply:
It’s a good idea to use `pathlib.Path` instead of playing around with strings. It’s more robust to OS differences

Reply:
@karlhigley I already resolved the issue using `pathlib.Path`. Have a look at it once more.